docs/section6/Rule6-6.md

```diff
@@ -37,4 +37,4 @@
 
           - Case 1: If lighting building area type is not specified: ```if NOT building_segment_p.lighting_building_area_type: FAIL```  
 
-          - Case 2: If lighting power does not equal to the value prescribed in Table 9.5.1 or lighting power in P_RMR is not the same as U_RMR: ```if ( ( interior_lighting_p.power_per_area == lighting_allowance_p ) OR ( interior_lighting_p == interior_lighting_u ) ): FAIL```  
+          - Case 2: If lighting power does not equal to the value prescribed in Table 9.5.1 or lighting power in P_RMR is not the same as U_RMR: ```if ( ( interior_lighting_p.power_per_area != lighting_allowance_p ) OR ( interior_lighting_p != interior_lighting_u ) ): FAIL```  
```

Comment:
I think this RDS has the same issue as 6-4. Unless I am missing something, I don't see how RCT can tell that the lighting "neither exists nor is submitted with design documents". I don't think specifying or not specifying lighting_building_area_type means that. Also, on some projects lighting may be designed for some tenant spaces but not others, so we can't use the global building_segment_p.lighting_building_area_type attribute to determine where lighting is and isn't specified.  See my comment for 6-4 regarding the proposed solution.